### PR TITLE
Implement an analog clock

### PR DESCRIPTION
**Pull request by ysbrandB**
Implements #1858 (https://github.com/saproto/saproto/issues/1858).
Now we actually will have a half decent way to tell the time in Proto!

image

### CODE REVIEW

**Comment by Menkveld-24** [comment]
I think its nice to give it the same offset to the right of the screen as the div above it

File: client/src/views/ProtubeScreen.vue [hunk 1]
```diff
@@ -108,6 +111,7 @@ import {
 import socket, { connectSocket } from "@/js/ScreenSocket";
 import YoutubePlayer from "youtube-player";
 import enums from "@/js/Enums";
+import AnalogClock from "../components/AnalogClock.vue";
```
Comment (Menkveld-24) [inline client/src/views/ProtubeScreen.vue:114]: I’d use @/components import instead

File: client/src/components/AnalogClock.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+const minute = ref(0);
+const hour = ref(0);
+const hourRotation = computed(() => {
+  return `transform:rotate(${hour.value * (360 / 12)}deg)`;
+});
+const minuteRotation = computed(() => {
+  return `transform:rotate(${minute.value * (360 / 60)}deg)`;
+});
+onMounted(() => {
```
Comment (Menkveld-24) [inline client/src/components/AnalogClock.vue:43]: Some white lines might be nice between the functions

**Comment by Jessseee** [comment]
I think that takes up way too much space on the screen. A simple digital clock would be sufficient and look way less cluttered
.

**Comment by ysbrandB** [comment]
> I think that takes up way too much space on the screen. A simple digital clock would be sufficient and look way less cluttered.

I think this is a nicer way than doing another white div with a time formatted as 00:00:00 as we already have one of those.
In use, it does not feel intrusive to me at all, and I personally like the aesthetic of an analogue clock more.

**Pull request merged**